src/metrics/prometheus.js

```diff
@@ -75,7 +75,11 @@ export default class PrometheusMetricsFactory {
       name = this._namespace + '_' + name;
     }
     if (!(key in this._cache)) {
-      this._cache[key] = new metric(name, help, labelNames);
+      this._cache[key] = new metric({
+        name: name,
+        help: help,
+        labelNames: labelNames,
+      });
```

Comment:
iirc this can be simplified to `{ name, help, labelNames }`, and Node will define hash keys equal to variable names.

Reply:
Yes I know it can be simplified.
But I just refered to existing code like [this](https://github.com/jaegertracing/jaeger-client-node/blob/master/src/span.js#L238) and make same.
It is not a big deal, I can change. :)